src/display.c

```diff
@@ -155,7 +160,15 @@ size_t LCUIDisplay_Render(void)
 		 */
 
 		/* Repaint dirty rectangles of surface */
+#ifdef USE_OPENMP
+#pragma omp parallel
+#pragma omp single
+#endif
 		for (LinkedList_Each(rn, &rects)) {
+#ifdef USE_OPENMP
+#pragma omp task firstprivate(rn)
+			printf("thread_num: %d\n", omp_get_thread_num());
```

Comment:
printf() takes a long time to call, you should remove it

Reply:
I removed `printf()` in d5cf404.